SPECS/Cython/Cython.spec

```diff
@@ -1,79 +1,396 @@
-%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-Summary:       C extensions for Python
-Name:          Cython
-Version:       0.28.5
-Release:       7%{?dist}
-Group:         Development/Libraries
-License:       ASL 2.0
-URL:           https://cython.org
-#Source0:      https://github.com/cython/cython/archive/%{version}.tar.gz
-Source0:       %{name}-%{version}.tar.gz
-Patch0:        fix_abc_tests.patch
-Vendor:        Microsoft Corporation
-Distribution:  Mariner
-BuildRequires: python2-devel
-BuildRequires: python2-libs
-BuildRequires: python-xml
-Requires:      python2
-
-%description
-Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). It makes writing C extensions for Python as easy as Python itself.
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%define python3_sitearch %(python3 -c "from distutils.sysconfig import get_python_lib; import sys; sys.stdout.write(get_python_lib(1))")
+
+%global upname cython
+
+Name:           Cython
+Version:        0.29.13
+Release:        6%{?dist}
+Summary:        Language for writing Python extension modules
+Vendor:         Microsoft
+Distribution:   Mariner
+License:        ASL 2.0
+URL:            https://www.cython.org
+#Source0:       https://github.com/%{upname}/%{upname}/archive/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+
+BuildRequires:  gcc
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-xml
+
+%global _description \
+Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (baded on Pyrex). It makes writing C extensions for Python as easy as Python itself.}
+
+%description %{_description}
+
+%package -n python3-%{name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{name}}
+Provides:       %{name} = %{version}-%{release}
+Provides:       %{name}%{?_isa} = %{version}-%{release}
+Obsoletes:      %{name} < %{version}-%{release}
+
+%description -n python3-%{name} %{_description}
+
+Python 3 version.
 
 %prep
-%setup -q -n cython-%{version}
-%patch0 -p1
+%setup -n %{upname}-%{version}
 
 %build
-python2 setup.py build
+python3 setup.py build
 
 %install
-python2 setup.py install --skip-build --root %{buildroot}
 
-%check
-# Skip lvalue_refs test which was fixed in a later release: https://github.com/cython/cython/pull/2783
-echo "lvalue_refs" >> tests/bugs.txt
-make %{?_smp_mflags} test
+python3 setup.py install --skip-build --root=%{buildroot}
+rm -rf %{buildroot}%{python3_sitelib}/setuptools/tests
 
-%clean
-rm -rf %{buildroot}
-
-%files
-%defattr(-,root,root,-)
+%files -n python3-%{name}
```

Comment:
It looks like this change is effectively removing the Photon's `Cython` package and adding a new `python3-Cython` package fom Fedora 31. While in core that shouldn't be a problem (I see you're already updating `python-lxml` to use the new package), I'm worried about derivatives, which stay with `Cython` as they will no longer receive any CVE fixes through no fault of their own unless they switch to the new package name. So I guess the question from my end is: do we know what is our policy for re-naming packages mid-release? Have you checked which (if any) derivatives depend on our current version of `Cython`? Fortunately, `Cython` doesn't seem to be a run-time requirement for anything and considering what this tool is for, I doubt derivatives make any use of it, but we must make sure.